frame/ddc-metrics-offchain-worker/src/lib.rs

```
@@ -297,8 +319,64 @@ impl<T: Trait> Module<T> {
         Ok(signer)
     }
 
+    fn sc_get_current_period_ms() -> ResultStr<u64> {
+        let contract_id = T::ContractId::get();
+        let contract_idx = <<T::CT as frame_system::Trait>::Lookup as StaticLookup>::lookup(contract_id).unwrap();
+
+        let call_data = Self::encode_get_current_period_ms();
+        let (exec_result, _gas_consumed) = pallet_contracts::Module::<T::CT>::bare_call(
+            Default::default(),
+            contract_idx,
+            0u32.into(),
+            100_000_000_000,
+            call_data,
+        );
+
+        let (data) = match exec_result {
+            Ok(v) => v.data,
+            Err(_) => [0].to_vec(),
```

I guess this means to use 0 as default. Then it must be 8 bytes.

Anyhow, I think this should return the error instead of a default.

---

ok. My missing, this question i already asked you here ( [url](https://cerenetwork.atlassian.net/browse/CBI-716?focusedCommentId=12116) )